modules/judges.py

```diff
@@ -83,9 +84,12 @@ def downloadSubmissions(self):
         self.clearSolutions()
         source_dowload_url = f"{self.rootUrl}{self.acSubmissionsUrl}"
         self.downloadUrl(source_dowload_url, self.s, self.zipUrl)
-        self.extractZip()
-        print("Solutions downloaded and extracted.")
-        self.moveGymSolToCF()
+
+        if self.extractZip():
+            print("Solutions downloaded and extracted.")
+            self.moveGymSolToCF()
+        else:
+            exit(1)  # If its a bad zip file then exiting
```

Comment:
checking if successfully extracted and extarctzip() function returned true then printing Solutions downloaded and extracted.
otherwise if extarctzip method already printed bad zip file and returned false then exiting